torch/csrc/jit/tensorexpr/tensor.h

```diff
@@ -113,6 +115,24 @@ class Tensor : KernelScopedObject {
   const Expr* initializer_{nullptr};
 };
 
+class CompoundTensor : public Tensor {
```

Comment:
`TORCH_API` as well?

Reply:
???? Maybe? TORCH_API still kind of baffles me. Better safe than sorry.